account_invoice_report_grouped_by_picking/views/report_invoice.xml

```diff
@@ -19,7 +19,16 @@
         <t t-if="picking != last_picking">
             <tr t-if="picking">
                 <td colspan="10">
-                    <strong t-esc="'Order: %s Picking: %s %s' % (picking.sale_id.name, picking.date_done[:10], picking.name)"/>
+                    <strong>
+                        <span>Order:</span>
+                        <span t-esc="picking.sale_id.name"/>
+                        <t t-if="picking.sale_id.client_order_ref">
+                            <span t-translation="off"> (</span><span t-esc="picking.sale_id.client_order_ref"/><span t-translation="off">)</span>
+                        </t>
+                        <span>Picking:</span>
+                        <span t-esc="picking.date_done[:10]" t-options="{'widget': 'date'}"/>
```

Comment:
Use t-field to haven't problems with timezone

Reply:
There's no timezone in only dates ;)